<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAutor</class>
 <widget class="QDialog" name="AddAutor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>152</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LBC - Add Author</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>12</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name, Lastname</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>30</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nationality</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>48</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Interests</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>66</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Chapters</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ImieNazw">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>12</y>
     <width>199</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Narod">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>30</y>
     <width>199</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Spec">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>48</y>
     <width>199</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Rozdz">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>66</y>
     <width>199</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>114</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>114</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>AddAutor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>AddAutor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
